ziti/cmd/ziti/cmd/config_templates/router.yml

```diff
@@ -0,0 +1,136 @@
+{{/*
+ Config Format Version
+
+ Whenever a breaking change is made to the semantics of this configuration file, the configuration version
+ expected by the router will be incremented. When the controller tries to load a configuration file with
+ an incompatible version, it will abort with a message prompting the operator to seek out the breaking changes
+ documentation.*/}}
+v: 3
+
+identity:
+  cert:                 "{{ .ZitiPKI }}/{{ .ZitiCtrlIntermediateName }}/certs/{{ .RouterName }}-client.cert"
+  server_cert:          "{{ .ZitiPKI }}/{{ .ZitiCtrlIntermediateName }}/certs/{{ .RouterName }}-server.cert"
+  key:                  "{{ .ZitiPKI }}/{{ .ZitiCtrlIntermediateName }}/keys/{{ .RouterName }}-server.key"
+  ca:                   "{{ .ZitiPKI }}/{{ .ZitiCtrlIntermediateName }}/certs/{{ .RouterName }}-cas.cert"
+
+ctrl:
+  endpoint:             tls:{{ .ZitiCtrlHostname }}:{{ .ZitiFabCtrlPort }}
+
+link:
+{{- if .IsPrivate }}
```

Comment:
is it easier to add the hashtag to each line if `.IsPrivate` instead of duplicating the blocks?

Reply:
Something like this?

`{{ if .IsPrivate }}#{{ end }}`